Controller/Comment.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * @author           Serge Kishiko <sergekishiko@gmail.com>
+ * @copyright        (c) 2015-2017, Pierre-Henry Soria. All Rights Reserved.
+ * @license          Lesser General Public License <http://www.gnu.org/copyleft/lesser.html>
+ * @link             http://hizup.uk
+ */
+
+namespace TestProject\Controller;
+
+class Comment extends Blog
+{
+    public function __construct()
+    {
+        $this->oUtil = new \TestProject\Engine\Util;
```

Comment:
If you done what I tell you in my previous above comment, you can just remove the constructor here, so PHP will automatically call the parent constructor